ci_pod/jupyterlab/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM fedora:38
+
+ARG JUPYTERLAB_VERSION
+ENV JUPYTERLAB_VERSION=$JUPYTERLAB_VERSION
+ARG MAINTAINER
+ENV MAINTAINER=$MAINTAINER
+
+RUN dnf update -y
+LABEL maintainer="Your Name <$MAINTAINER>"
+
+# Install system dependencies
+RUN dnf update -y && \
+    dnf install -y wget podman && \
+    dnf install -y curl && \
+    dnf install -y python3 python3-pip python3-devel nodejs npm && \
+    dnf clean all && \
+    rm -rf /var/cache/dnf
+
+# Install JupyterLab and widgets
+RUN pip3 install --no-cache-dir jupyterlab=="$JUPYTERLAB_VERSION" && \
+    jupyter labextension install @jupyter-widgets/jupyterlab-manager
+
+# Install benchmark-runner
+RUN pip3 install --no-cache-dir benchmark-runner
```

Comment:
Combine all of the RUN statements into one, and use microdnf with fedora-minimal.  Each separate RUN statement creates a separate layer in the image.

Reply:
Done, 
Thx for your comment to combined all the statement.
Regarding the image I prefer to use fedora and not minimal because we are using it through jupyterlab terminal